src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -394,6 +435,33 @@ export const selectAccountIdsByAssetId = createSelector(
   findAccountsByAssetId
 )
 
+export const selectAccountIdsByAssetIdAboveBalanceThreshold = createDeepEqualOutputSelector(
+  selectPortfolioAccounts,
+  selectAssetIdParam,
+  selectPortfolioFiatAccountBalances,
+  selectBalanceThreshold,
+  (portfolioAccounts, assetId, accountBalances, balanceThreshold) => {
+    const accounts = findAccountsByAssetId(portfolioAccounts, assetId)
+    const aboveThreshold = Object.entries(accountBalances).reduce<AccountSpecifier[]>(
```

Comment:
Inline

Reply:
Would you please elaborate more?

Reply:
Not sure I got this either, this is just a single line although it can look from the diff on GitHub as they're two lines

Reply:
We can return the result of `aboveThreshold` directly, rather than assigning to a `const` first - but assigning can also be useful for debugging, so it's fine - resolving.